client/src/components/reader/Annotation/Annotatable.js

```diff
@@ -304,9 +300,9 @@ class Annotatable extends Component {
         : null}
 
         { this.props.currentUser ?
-          <Annotation.Popup
+          <Annotation.Popup.Wrapper
             currentUser={this.props.currentUser}
-            share={this.shareSelection}
+            shareUrl={`/read/${this.props.projectId}/section/${this.props.sectionId}`}
```

Comment:
I haven't tried branch yet, but just eyeballing it this looks wrong. I believe the path to a section contains the text segment, not the project segment.